tdrs-backend/tdpservice/search_indexes/tasks.py

```diff
@@ -0,0 +1,37 @@
+"""Shared celery search_indexes tasks for beat."""
+
+from __future__ import absolute_import
+import time
+from tdpservice.users.models import User
+from celery import shared_task
+from tdpservice.core.utils import log
+from django.core.management import call_command
+
+
+@shared_task
+def reindex_elastic_documents():
+    """Run the django-elasticsearch-dsl management command to reindex the elastic instance."""
+    system_user, created = User.objects.get_or_create(username='system')
+    if created:
+        log('Created reserved system user.')
+
+    start = time.time()
+    log('Starting elastic reindex.', {
+        'user_id': system_user.pk,
+        'object_id': None,
+        'object_repr': ''
+    })
+
+    call_command('tdp_search_index', '--rebuild', '--use-alias', '--parallel', '-f')
```

Comment:
I think to avoid the weird name datetime appending issue we discussed, we should consider using `subprocess`.  I made a patch for you to check out. This method also keeps the console logs a lot cleaner.
[elastic.patch](https://github.com/raft-tech/TANF-app/files/14621401/elastic.patch)